src/main.js

```diff
@@ -9,13 +9,16 @@ import { Results } from "./components/results/results-screen.js";
 import { Select } from "./components/select/select-screen.js";
 import { HowToPlay } from "./components/how-to-play.js";
 import { Game } from "./components/game.js";
+import { Shop } from "./components/shop/shop.js";
 import { Pause } from "./components/overlays/pause.js";
 import { settingsChannel } from "./core/settings.js";
 import { eventBus } from "./core/event-bus.js";
 import { startup } from "./core/startup.js";
+
 //TODO Re-enable once Phaser plugins are built (should be in 3.24)
 //import "/node_modules/phaser/plugins/spine/dist/SpinePlugin.min.js";
 import "../lib/SpinePlugin.min.js"; //CAN BE REMOVED IF NOT USING SPINE
+import "../lib/rexuiplugin.min.js"; // CAN BE REMOVED IF NOT USING SHOP
```

Comment:
could this be imported in the shop so it only gets used when shop component is imported?

Reply:
I'll try that but I suspect it needs to be imported before game instantiation as it's a plugin (see get-default-plugins.js.)

Reply:
We should probably talk about the ES6 module thing - will try to find a window tomorrow. If it was an ES6 module it would be easier I think as we support zero build loading- I think right now you are possibly using our global loader to do this (I haven't looked though ) which would mean it has to be loaded up front. There's Phaser examples of loading a plugin in the scene's preloader so in theory this should be possible.

Reply:
Yes, let's talk further. For now, 4e4fd72 demonstrates we can happily move the import to the Shop screen and de-bloat the bundle somewhat for non-Shop games (as Adam suggested.)